README.md

```diff
@@ -1 +1,52 @@
-# Ademola-Portfolio
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
```

Comment:
- [ ] [REQUIRED] It is a professional practice to also add a project description after adding the project name. This will make your README look more professional.